src/Api/Models/Response/SecretResponseModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.Models.Response
```

Comment:
Let's move this to `src/Api/SecretManagerFeatures/Models/Response/SecretResponseModel.cs`. I want to avoid adding even more files to the generic Models directory.

Reply:
resolved in https://github.com/bitwarden/server/pull/2163/commits/1373b497db321fb4dd103995485c222650257138